stacks/analytics-ingest-lambda.yml

```diff
@@ -63,6 +71,24 @@ Resources:
                   - "ssm:GetParametersByPath"
                 Resource:
                   - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prx/${EnvironmentTypeAbbreviation}/analytics-*"
+        - PolicyName: KinesisWritePolicy
```

Comment:
I'd consider making a second IAM policy, since the policy needs of the various lambdas are diverging quite a bit